snippets/es6-syntax.cson

```diff
@@ -0,0 +1,17 @@
+'.source.coffee':
+
+  'ES6 import syntax':
+    'prefix': 'es.imp'
```

Comment:
I think there should be separate prefixes `es.imp` -> `import-ember` and `es.imp` -> `import-data`